modules/nw-sriov-example-vf-function-in-pod.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple-networks/configuring-sr-iov.adoc
+
+[id="example-vf-use-in-pod_{context}"]
+= Example use of a virtual function (VF) in a Pod
+
+You can run a remote direct memory access (RDMA) or a Data Plane Development Kit (DPDK) application in a Pod with SR-IOV VF attached.
+In the following example, a Pod is using a VF in RDMA mode:
```

Comment:
Should this be +This example shows a pod using a VF in RDMA mode:_
The current construct is a but awkward and changing would make it parallel the next example.

Reply:
That's much better. Great suggestion.